CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## @mercurial-finance/dynamic-amm-sdk [1.2.0] - PR[#196](https://github.com/mercurial-finance/mercurial-dynamic-amm-sdk/pull/196)
```

Comment:
@meteora-ag instead of @mercurial-finance

Reply:
@McSam94 The org is still not updated for package version 1.2.0

Reply:
ahh sorry, updated.